curriculum/challenges/english/25-front-end-development/review-typescript/6724e296dceca21b82426229.md

```diff
@@ -9,6 +9,238 @@ dashedName: review-typescript
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## What is TypeScript
+
+- **JavaScript**: JavaScript is a dynamically-typed language. This means that variables can receive any values at run time. The challenge of a dynamically-typed language is that the lack of type safety can introduce errors. 
```

Comment:
let's add an example here for reminder.
something like `let data = some random string;` then change it to `data = some random object or whatever`. 
Just to drive the point home further